src/game_heap.c

```diff
@@ -87,7 +82,7 @@ TbBool setup_heap_manager(void)
     const char* fname = prepare_file_path(FGrp_StdData, "creature.jty");
 #endif
     //TODO CREATURE_SPRITE Use rewritten file handling when reading is rewritten
```

Comment:
These todo's reference the change you just made right?

Reply:
oh missed those, I'll remove them